frame/dex/src/tests.rs

```diff
@@ -401,15 +401,17 @@ fn quote_price_should_work() {
 			false
 		));
 
-		assert_eq!(Dex::quote_price(None, Some(2), 3000), Some(60));
+		assert_eq!(Dex::quote_price_exact_tokens_for_tokens(None, Some(2), 3000, false), Some(60));
+		// Check it still gives same price:
+		assert_eq!(Dex::quote_price_exact_tokens_for_tokens(None, Some(2), 3000, false), Some(60));
```

Comment:
this is the same as the previous line. what could cause it to return a different price? do we need to introduce some delay first?